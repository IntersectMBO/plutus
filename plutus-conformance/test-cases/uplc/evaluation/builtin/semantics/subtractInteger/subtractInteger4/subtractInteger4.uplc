(program 1.0.0 [ [ (builtin subtractInteger) (con integer 0)] (con integer -327893248793249782347891) ])