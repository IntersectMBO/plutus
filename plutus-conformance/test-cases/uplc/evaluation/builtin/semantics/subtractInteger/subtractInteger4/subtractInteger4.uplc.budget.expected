({cpu: 322389
| mem: 603})